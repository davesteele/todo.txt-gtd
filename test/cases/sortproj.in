# BProj
btask @computer +BProj

# AProj
atask @computer +AProj

# _None
