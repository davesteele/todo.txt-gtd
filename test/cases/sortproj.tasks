
@computer
btask @computer +BProj
atask @computer +AProj
