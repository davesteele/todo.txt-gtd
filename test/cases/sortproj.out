# AProj
atask @computer +AProj

# BProj
btask @computer +BProj

# _None
